ui/components/NotificationCenter/metadata.js

```diff
@@ -105,35 +94,37 @@ const ComponentWithIcon = ({ component }) => {
   const kind = Metadata.toLowerCase();
 
   const paths = [
-    `ui/public/static/img/meshmodels/${modelname}/color/${modelname}-color.svg`,
-    `ui/public/static/img/meshmodels/${modelname}/white/${modelname}-white.svg`,
     `ui/public/static/img/meshmodels/${modelname}/color/${kind}-color.svg`,
     `ui/public/static/img/meshmodels/${modelname}/white/${kind}-white.svg`,
+    `ui/public/static/img/meshmodels/${modelname}/color/${modelname}-color.svg`,
+    `ui/public/static/img/meshmodels/${modelname}/white/${modelname}-white.svg`,
   ];
 
-  const [finalPath, setFinalPath] = useState(
-    'ui/public/static/img/meshmodels/meshery-core/color/meshery-core-color.svg',
-  );
+  const defaultPath = 'ui/public/static/img/meshmodels/meshery-core/color/meshery-core-color.svg';
+
+  const [finalPath, setFinalPath] = useState(defaultPath);
 
   useEffect(() => {
     const loadImages = async () => {
-      if (imageCache[modelname]) {
-        setFinalPath(imageCache[modelname]);
-        return;
```

Comment:
Why is this caching removed?

Reply:
It was causing the last cached image for the entire model